README.md

```diff
@@ -43,11 +44,11 @@ and [this link](https://packagist.org/providers/php-http/async-client-implementa
 
 #### OPTIONAL DEPENDENCIES
 
-#### 1.) Install PHP [ext-grpc](https://pecl.php.net/package/gRPC)
+#### 1) Install PHP [ext-grpc](https://pecl.php.net/package/gRPC)
 
 **The PHP gRPC extension is only needed, if you want to use the OTLP GRPC Exporter.**
 
-There are basically three ways to install the gRPC extension which will be described below. Keep in mind, that whatever way
+The three ways to install the gRPC extension are described below. Keep in mind, that whatever way
```

Comment:
If you want to get rid of the "basically", please just say "Three ways to..."
(There could be more than three ways in theory)